File: MapGenerator/DB/Boundary.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MapGenerator.DB
{
    public class Boundary
    {
        public int X1 { get; set; }
        public int Y1 { get; set; }
        public int X2 { get; set; }
        public int Y2 { get; set; }

        [NotMapped]
        public int Width
        {
            get => X2 - X1;
        }

        [NotMapped]
        public int Height
        {
            get => Y2 - Y1;
        }
    }
}

File: AL-MapViewer/db/Boundary.js
class Boundary
{
    X1 = 0;
    Y1 = 0;
    X2 = 0;
    Y2 = 0;

    get [Symbol.toStringTag]() {
        return `(${this.X1}, ${this.Y1}, ${this.Width}, ${this.Height})`;
    }

    get Width()
    {
        return this.X2 - this.X1;
    }

    get Height()
    {
        return this.Y2 - this.Y1;
    }

}

module.exports = {Boundary};